cfgov/unprocessed/css/molecules/translation-links.less

```diff
@@ -1,5 +1,10 @@
 @import (reference) 'cfpb-core.less';
 
+//Fix share&print icons not right aligning on some pages
+html[lang='ar'] .u-layout-grid_main {
+  direction: rtl;
+}
+
```

Comment:
Interesting, mmm, yes we probably should do this within the major CSS grid layout sections overall, however, a few notes for this:

- MDN has a warning about doing directionality in CSS (see https://developer.mozilla.org/en-US/docs/Web/CSS/direction)

> Warning: Where possible, authors are encouraged to avoid using the direction CSS property and use the HTML [dir](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes#dir) global attribute instead.

We should follow this advice I think, since we have `lang` in the markup and other instances of `dir` in the markup.

- Were this to be in the CSS, it should appear with the other layout CSS, namedly https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/unprocessed/css/enhancements/layout-base.less (or `layout-2-1.less` or `layout-1-3.less`).

For now, we should probably add `{% if language=='ar' %} dir='rtl' {% endif %}` to https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/v1/jinja2/v1/includes/molecules/social-media.html#L71 (this could also be done in https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/v1/jinja2/v1/layouts/layout-2-1-rtl.html#L16, but maybe we want to scope this to `m-social-media` while we have other instances of `dir='rtl'` on these pages).

Reply:
This is no longer needed with your "dir=rtl" update (7807), thanks Ans!